log/controller.log

```diff
@@ -153,3 +153,394 @@ info🏷️: [Feb-25-2022 08:41:45]: Cleared Sandbox for next simulation
 info🏷️: [Feb-25-2022 08:41:45]: Stored simulation details to simulations/7d1563a6-e015-46b3-a260-3df5db378ae2/6d67e260-92da-4f27-87b6-7024f6efbc8b/3
 info🏷️: [Feb-25-2022 08:41:45]: Step 3 finished execution
 
+info🏷️: [Feb-25-2022 09:40:45]: test1.txt,test 1
```

Comment:
I think we can ignore the issue now, but the log folder should probably not be pushed to git.